access/resource_secret_scope.go

```diff
@@ -150,4 +153,20 @@ func ResourceSecretScope() *schema.Resource {
 			return NewSecretScopesAPI(ctx, c).Delete(d.Id())
 		},
 	}.ToResource()
+
+	resource.CustomizeDiff = customdiff.Sequence(
+		func(ctx context.Context, diff *schema.ResourceDiff, v interface{}) error {
+			kvLst := diff.Get("keyvault_metadata").([]interface{})
+			if len(kvLst) == 0 {
+				return nil
```

Comment:
is it still possible to create databricks scopes?

Reply:
yes